examples/simple_calculator/README.md

```diff
@@ -276,8 +277,7 @@ general:
   ```
 
 ### Access the request metadata
-Get the instance of the {class}`aiq.builder.context.AIQContext` object using the {method}`aiq.builder.context.AIQContext.get()` method. This will give you access to the metadata
-method which holds the request attributes defined by the user on request.
+Get the instance of the {class}`aiq.builder.context.AIQContext` object using the {method}`aiq.builder.context.AIQContext.get()` method. This will give you access to the metadata method which holds the request attributes defined by the user on request. A complete example of the function can be found in `src/aiq/tool/server_tools.py.`
```

Comment:
I would remove the Sphinx markup as the examples are not included in the Sphinx builds and this won't render into something useful for the user.

```suggestion
Get the instance of the `aiq.builder.context.AIQContext` object using the `aiq.builder.context.AIQContext.get()` method. This will give you access to the metadata method which holds the request attributes defined by the user on request. A complete example of the function can be found in `src/aiq/tool/server_tools.py.`
```